src/main/java/com/ibm/crypto/plus/provider/EdDSAPrivateKeyImpl.java

```diff
@@ -90,13 +90,13 @@ public EdDSAPrivateKeyImpl(OpenJCEPlusProvider provider,
             }
 
             if (this.key == null) {
-                int pub_size = CurveUtil.getPublicCurveSize(curve);
+                int pub_size = CurveUtil.getCurveSize(curve);
```

Comment:
Should we change the name of the variable to `keySize`?

Reply:
Given that the public and private keys are equivilant I am ok using `keySize` as the variable name here.